module/MotionParser.h

```diff
@@ -51,6 +52,7 @@ enum class COMMAND {
   CM,  // 交点サークルから直線の中点
   PR,  // Pwm値指定回頭
   ST,  // 左右モーターストップ
+  BT,
```

Comment:
```suggestion
  BT, // ブロック投げ入れ
```
処理に影響はないですが、他のところが書いてあるので。